### DISCUSSION TITLE
i3 4.22 released!

### DISCUSSION THREAD

**Discussion by stapelberg** [i3/i3#5342] [Announcements]
Release notes can be found at https://i3wm.org/downloads/RELEASE-NOTES-4.22.txt

**Comment by wmerfalen** [comment]
Were there any security vulnerability fixes in this release?

**Reply by stapelberg** [reply]
No.

**Reply by Cu34v0** [reply]
What security issues are there? I was unaware that it had vulnerabilities

**Reply by stapelberg** [reply]
There are no known security issues in i3.

**Reply by Cu34v0** [reply]
Ok thanks for the great work i3wm 👍🏼

**Comment by ricardoseriani** [comment]
I don't know if this is the correct place to say this, but Debian and Ubuntu repositories are not updated to i3-wm version 4.22.
I tried to open an issue, but the bot closed because it detected that I'm not running the latest  version (this is exactly the problem that I'm trying to report 😅)

**Reply by stapelberg** [reply]
Which repository specifically are you using?

The repositories at baltocdn.com are updated automatically.

The repository at debian.sur5r.net will be updated when sur5r gets around to it (manually).

**Reply by ricardoseriani** [reply]
I only knew about debian.sur5r.net

There are any instructions on how to use baltocdn.com ?

It's any difference between the two, except for one is automatically updated, and the other is manually ?

**Reply by stapelberg** [reply]
All repositories are documented at https://i3wm.org/docs/repositories.html.

Yes, the difference is that the automatically built ones are nightly versions, whereas the sur5r repository contains released stable versions of i3 only.

**Reply by ricardoseriani** [reply]
Oh, I see...
The baltocdn.com it's the development release repository.
Thanks for the information and kudos for merging i3-gaps into i3.

**Comment by anthony-S93** [comment]
`i3-wm 4.22` is now on Arch Linux's `community` repo. `i3-wm 4.22` is currently running on my system as a drop-in replacement for `i3-gaps` without any issues. 

Thank you developers for all your hard work.

**Reply by Cightline** [reply]
Another thank you for merging i3-gaps.

**Comment by gardotd426** [comment]
@FinnHerring is the i3-gaps-rounded project going to be moved over to using upstream i3wm as its base, since i3-gaps is dead and is no longer maintained?

**Reply by okraits** [reply]
You  have to ask the developer of that project.

**Reply by jbenden** [reply]
I guess I need to look into merging/sorting this whole mess soon... Thanks!

**Reply by jbenden** [reply]
Merged.
